packages/algob/src/builtin-tasks/sign-multisig.ts

```diff
@@ -0,0 +1,97 @@
+import algosdk from "algosdk";
+import fs from "fs-extra";
+import path from "path";
+
+import { task } from "../internal/core/config/config-env";
+import { ASSETS_DIR } from "../internal/core/project-structure";
+import { RuntimeEnv } from "../types";
+import { TASK_SIGN_MULTISIG } from "./task-names";
+
+export interface TaskArgs {
+  file: string
+  account: string
+  out: string
+}
+
+async function multiSignTx (
+  taskArgs: TaskArgs,
+  runtimeEnv: RuntimeEnv
+): Promise<void> {
+  const accounts = runtimeEnv.network.config.accounts;
+  let flag = false;
+  const signer = {
+    addr: "",
+    sk: new Uint8Array(0)
+  };
+  let signedTxn;
+  for (var account of accounts) {
+    if (account.name === taskArgs.account) {
+      flag = true;
+      signer.addr = account.addr;
+      signer.sk = account.sk;
+    }
+  }
+  if (!flag) {
+    console.error("No account with the name \"%s\" exists in the config file.", taskArgs.account);
+    return;
+  }
+  const p = path.join(ASSETS_DIR, taskArgs.file);
+  const txFile = fs.readFileSync(p);
+  if (txFile === undefined) {
+    throw new Error(`File ${taskArgs.file} does not exist`);
+  }
+  console.log(txFile);
+  const tx = algosdk.decodeObj(txFile);
+  if (!tx.blob) {
+    console.error("The decoded transaction doesn't appear to be a signed transaction.");
```

Comment:
That doesn't make sense as that will be done while executing the script. The output of the script will then be used as an input here. 

Because I can only extract the `mparams` required to sign the transaction after deciding it's blob.